proto/public.proto

```diff
@@ -294,6 +294,8 @@ message StatSummaryRequest {
     Resource to_resource   = 4;
     Resource from_resource = 5;
   }
+
+  bool skip_stats = 6;  // true if we want to skip stats from Prometheus (used in the dashboard sidebar)
```

Comment:
I think we can remove `(used in the dashboard sidebar)` from the comment, since the comments in the proto tend to be descriptions of what the fields signify